### DISCUSSION TITLE
在 Alfred 中发送内容到 flomo

### DISCUSSION THREAD

**Discussion by lightory** [flomoapp/3rd-party-tools#5] [分享创造]
https://blog.flomoapp.com/58.html

**Comment by moyuguy** [comment]
其实可以在workflow的Workflow Environment Variables设置apikey

**Reply by levie-vans** [reply]
对的

**Reply by iSteveyang** [reply]
我改进了一下，适用于Alfred3+可以试试
https://github.com/iSteveyang/Send2FlomoNG-Alfred-workflow
